Where_Is_My_Wife/Assets/_Main/Scripts/UI/MenuManager.cs

```diff
@@ -59,16 +66,17 @@ await menuToMove.DOMove(finalPosition, _menuAnimationDuration)
             _eventSystem.enabled = true;
             nextButton.Select();
         }
+
+        private void ChangeToLastPlayedStoryModeLevelBackground()
+        {
+            ChangeBackground(DataSaveManager.Instance.GetData<int>(DataSaveManager.LastPlayedStoryModeLevelIndexKey));
+        }
         
-        public async UniTaskVoid CloseMenu()
+        public void ChangeBackground(int index)
```

Comment:
Rename to `levelNumber` and add the `-1` on the  `[]`

Reply:
I can rename to levelNumber, but not the -1.